.github/workflows/build_and_deploy_search_bar.yml

```diff
@@ -18,12 +18,28 @@ jobs:
       build_script: build-search-bar-only
       cloud_region: EU
 
-  call_misc_tests:
-    uses: ./.github/workflows/miscellaneous_tests.yml
+  merge_multiple_us_artifacts:
+    uses: ./.github/workflows/merge_upload_multiple_artifacts.yml
+    needs:
+      - call_build
+    with:
+      cloud_region: US
+
+  merge_multiple_eu_artifacts:
+    uses: ./.github/workflows/merge_upload_multiple_artifacts.yml
+    needs:
+      - call_build_for_eu
+    with:
+      cloud_region: EU
 
   call_acceptance_search_bar:
     uses: ./.github/workflows/acceptance_search_bar.yml
-    needs: call_build
+    needs: 
+      - merge_multiple_us_artifacts
+      - merge_multiple_eu_artifacts
```

Comment:
we don't need any of the changes to this file, right? and same with build_and_deploy_hold?